src/metabase/api/persist.clj

```diff
@@ -83,15 +94,15 @@
   "Set the interval (in hours) to refresh persisted models. Shape should be JSON like {hours: 4}."
   [:as {{:keys [hours], :as _body} :body}]
   {hours HoursInterval}
-  (api/check-superuser)
+  (validation/check-has-application-permission :setting)
```

Comment:
It'd be nice to add some permission tests into `metabase-enterprise.advanced-permissions.api.setting-test`